x/distribution/keeper/querier.go

```diff
@@ -207,7 +207,17 @@ func queryDelegationRewards(ctx sdk.Context, _ []string, req abci.RequestQuery,
 	ctx, _ = ctx.CacheContext()
 
 	val := k.stakingKeeper.Validator(ctx, params.ValidatorAddress)
+	if val == nil {
+		// TODO: Should use ErrNoValidatorFound from staking/types
```

Comment:
We should be using those error types defined in `staking`. I see a few options:

1. Simply import and use `staking/types`
2. Add alias functions in `distribution` which are set at runtime by the `staking` module
3. Have methods in the `staking` keeper actually return an appropriate error instead of `nil` or a `bool`

/cc @rigelrozanski @cwgoes

Reply:
+1 to option `(3)`